.buildkite/auditbeat/auditbeat-pipeline.yml

```diff
@@ -48,6 +48,17 @@ steps:
           - github_commit_status:
               context: "auditbeat: check/update"
 
+      - label: "Auditbeat: Run pre-commit"
+        command: "pre-commit run --all-files"
+        agents:
+          image: "docker.elastic.co/ci-agent-images/platform-ingest/buildkite-agent-beats-ci-with-hooks:latest"
+          cpu: "4000m"
```

Comment:
is 4 full cpu cores an overkill for running this test? Have you tested it with e.g. 2, is it enough?

Reply:
What if we omit the cpu and mem limits?